### PR TITLE
docs(CONTRIBUTING): fix the link to bugs label

### PR DESCRIPTION
**Pull request by liutiming**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ If you want some code examples for how to write watchers or other types of clien
 There are many ways to contribute to ActivityWatch:
 
  - Work on issues labeled [`good first issue`][good first issue] or [`help wanted`][help wanted], these are especially suited for new contributors.
- - Fix [`bugs`][bugs].
+ - Fix [`bugs`][bug].
```
Comment (johan-bjareholt) [inline CONTRIBUTING.md:31]: "bug" should still be in plural I think

Comment (liutiming) [reply CONTRIBUTING.md:31]: the issue tag is in singular, tho

Comment (ErikBjare) [inline CONTRIBUTING.md:31]:
```suggestion
 - Fix [`bugs`][bugs].
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ Most of the above will get you up on our [contributor stats page][contributors]
 
 [good first issue]: https://github.com/ActivityWatch/activitywatch/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
 [help wanted]: https://github.com/ActivityWatch/activitywatch/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22
-[bugs]: https://github.com/ActivityWatch/activitywatch/issues?q=is%3Aissue+is%3Aopen+label%3A%22bugs%22
+[bug]: https://github.com/ActivityWatch/activitywatch/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22
```
Comment (ErikBjare) [inline CONTRIBUTING.md:45]:
```suggestion
[bugs]: https://github.com/ActivityWatch/activitywatch/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22
```

**Pull request merged**